app/models/user.rb

```diff
@@ -38,7 +38,7 @@ class User < ActiveRecord::Base
   def role?(base_role)
     return false unless role
 
-    ROLES.index(base_role) <= ROLES.index(role.to_sym)
+    ROLES.index(base_role) == ROLES.index(role.to_sym)
```

Comment:
How do we want to deal with invalid roles? It is a getter function, so we cannot get into an invalid state with an invalid role given. With an invalid role it will always return false (which is correct), so we dont actually have a "real" error.
However, throwing an error might make it easier to find bugs